src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -222,7 +222,8 @@ private static async Task SetAutoCompleteAsync(GitPullRequest pullRequest)
                     new IdentityRefWithVote { Vote = (short)Vote.Approved },
                     repository.Id,
                     pullRequest.PullRequestId,
-                    VSLSnapUserId.ToString()
+                    VSLSnapUserId.ToString(),
+                    cancellationToken : cancellationToken
```

Comment:
nit: formatting (no rush, just would be good to fix up in a future PR)

Reply:
what's the right format?